src/controllers/countries.js

```
@@ -0,0 +1,13 @@
+const { getCountries } = require('../services/countries');
+
+const fetchCountries = async (req, res) => {
+  try {
+    const countries = await getCountries();
+    res.json(countries);
+  } catch (error) {
+    console.error(error);
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)